packages/core-types/src/types/ICredentialIssuer.ts

```diff
@@ -12,13 +12,10 @@ import { IKeyManager } from './IKeyManager.js'
 import { IIdentifier, IKey } from "./IIdentifier.js";
 import { UsingResolutionOptions } from './ICredentialVerifier.js'
 
-/**
- * The type of encoding to be used for the Verifiable Credential or Presentation to be generated.
- *
- * Only `jwt` and `lds` is supported at the moment.
- *
- * @public
- */
+/*
+*
+* @public
+*/
 export type ProofFormat = 'jwt' | 'lds' | 'EthereumEip712Signature2021' | 'bbs'
```

Comment:
still missing the `string` type, to make it extensible
```suggestion
export type ProofFormat = 'jwt' | 'lds' | 'EthereumEip712Signature2021' | 'bbs' | string
```